wasswa-derick/NairobiJavaGeeks
# [Feature #155965831] Display List of Github Users

#### What does this PR do?
* Display List of Github Users

#### Description of Task to be completed?
* Define the adapter to retrieve data from the data set and generate View objects based on that data. 
* Use the Adapter and RecyclerView in the MainActivity to ensure that I display the List of Users fetched from the GitHubAPI.

#### How should this be manually tested?
* Clone the project, ```git clone https://github.com/wasswa-derick/NairobiJavaGeeks.git```.
* In your terminal, ```cd NairobiJavaGeeks```
* Checkout to the ```ft-display-github-users-155965831``` branch.
* Build the project and run the application on a device or emulator.
* The application should launch an activity that renders a list of GitHub users as fetch from the GitHubAPI.

#### What are the relevant pivotal tracker stories?
[#155965773](https://www.pivotaltracker.com/story/show/155965773)
[#155965831](https://www.pivotaltracker.com/story/show/155965831)

#### ScreenShots

![lists](https://user-images.githubusercontent.com/39955231/46086215-4d4b8580-c1b0-11e8-93c0-4af7960f9316.png)
